### PR TITLE
[AV2-640]/unassigned review msg to mentor

### PR DESCRIPTION
**Pull request by ronakamani**

### CODE REVIEW

File: src/app/assessment/assessment.component.ts [hunk 1]
```diff
@@ -250,6 +250,21 @@ export class AssessmentComponent extends RouterEnter {
               ]
             });
           }
+          if (this.action === 'review' && !this.doReview) {
```
Comment (shawnm0705) [inline src/app/assessment/assessment.component.ts:253]: This is not the correct condition for this situation, if action === 'review' and !this.doReview that means the user is seeing the review.

Comment (ronakamani) [reply src/app/assessment/assessment.component.ts:253]: @shawnm0705 should I just add the role to the condition?

Comment (shawnm0705) [reply src/app/assessment/assessment.component.ts:253]: It’s nothing about checking the mentor role or not

The scenario is that the mentor is not assigned to the review, but he goes to the review page

The correct logic is
- API should not return the review data
- Front-end should pop up toast message if there’s no review data

File: src/app/assessment/assessment.component.ts [hunk 2]
```diff
@@ -331,6 +331,21 @@ export class AssessmentComponent extends RouterEnter {
         }
       },
       (error) => {
+        if (!this.review.status) {
```
Comment (shawnm0705) [inline src/app/assessment/assessment.component.ts:334]: This is still not the right place to put these codes. 
I'm working on [AV2-646] which will change the code structure of this part, so I think it's better to hold this story for now and go back to it after [AV2-646] has been merged.

Comment (ronakamani) [reply src/app/assessment/assessment.component.ts:334]: ok, sure. thanks @shawnm0705

**Comment by shawnm0705** [comment]
The graphql implementation as been merged to develop branch so that we can re-start this story now

**Comment by Barry-Chen-Intersective** [comment]
It need some changes to the develop branch. While we unassigns a mentor and access the review the direct link, the page display as an empty submission rather than a review.

**Pull request closed without merging**